README.md

```diff
@@ -13,17 +13,23 @@ To use the latest stable version of the Solidity compiler via Node.js you can in
 npm install solc
 ```
 
-### Using on the command line
+Alternatively, in order to get the latest development version, you can install the version available here on GitHub:
```

Comment:
Alright, I'll remove it. I forgot why I put it in but I think it was because it wasn't available on npm at the time I was writing these docs.